packages/devtools_app/lib/src/screens/inspector/inspector_controller.dart

```diff
@@ -443,10 +444,11 @@ class InspectorController extends DisposableController
     int subtreeDepth = 2,
   }) async {
     assert(!_disposed);
-    if (_disposed) {
+    if (_disposed || _treeGroups == null) {
       return;
     }
 
+    final treeGroups = _treeGroups!;
```

Comment:
put this before line 447, and then check `treeGroups == null` in the if so that we don't have to null asset twice

Reply:
Thanks! Done.